### ISSUE TITLE
Automate the editing of the distribution README

### ISSUE THREAD

**Issue by rdeltour** [daisy/pipeline#512]
The README file of the distribution contains information that needs to be updated on each release:

- the information number in the title
- the date in the title
- the release notes
- the link to the release page (based on the version number)

It would be better to automate these bits. The date and version number can be set directly at build-time using Maven filtering. The release notes could be listed in a separate file (eg CHANGES.txt) and injected at build time too.

**Comment by bertfrees** [comment]
Good idea. I also think that it is better in general to maintain release notes in the source code instead of on Github. On Github you can make links to the CHANGES.txt file. This is how I have been doing it for the braille modules and other projects. See https://github.com/daisy/pipeline-mod-braille/releases.

Some benefits of this approach are described here: http://www.mehdi-khalili.com/better-git-release-notes. Another benefit I just thought of is that in theory people, if they would like, can make pull requests for the release notes just like they would do for any other code.

**Comment by bertfrees** [comment]
I've added a "NEWS" file to keep the release notes of all releases. (I didn't call it "CHANGES" because that is usually used for keeping a continuous log of changes, not grouped per release.)
